test/DynamoCoreWpfTests/NodeAutoCompleteSearchTests.cs

```diff
@@ -255,6 +255,40 @@ public void NodeSuggestions_DefaultSuggestions()
             var searchViewModel = ViewModel.CurrentSpaceViewModel.NodeAutoCompleteSearchViewModel;
             searchViewModel.PortViewModel = inPorts[0];
 
+            var suggestions = searchViewModel.GetMatchingSearchElements();
+            Assert.AreEqual(6, suggestions.Count());
+
+            var suggestedNodes = suggestions.Select(s => s.FullName);
+            var expectedNodes = new[] { "DSCoreNodes.DSCore.Color.Add",
+                "DSCoreNodes.DSCore.Color.ByARGB",
+                "DSCoreNodes.DSCore.Color.Divide",
+                "DSCoreNodes.DSCore.Color.Multiply",
+                "DSCoreNodes.DSCore.ColorRange.GetColorAtParameter",
+                "DSCoreNodes.DSCore.IO.Image.Pixels"};
+
+            for (int i = 0; i < 6; i++)
+            {
+                Assert.AreEqual(expectedNodes.ElementAt(i), suggestedNodes.ElementAt(i));
+            }
```

Comment:
Maybe use https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.sequenceequal?view=net-5.0

Reply:
There are some other alternatives in https://stackoverflow.com/questions/37020277/c-sharp-how-to-check-if-list-of-items-are-in-alphabetical-order-or-not.

Reply:
updated